src/main/java/io/qameta/allure/bamboo/AllureBuildCompleteAction.java

```diff
@@ -116,15 +131,50 @@ public void execute(@NotNull Chain chain, @NotNull ChainResultsSummary chainResu
         }
     }
 
-    private void prepareResults(List<File> artifactsTempDirs, Chain chain, ChainExecution chainExecution) throws IOException, InterruptedException {
+    /**
+     * I'm a friend, and it's a gift for all you that wants to customize allure report for your projects. :)
+     * @param allureReportDir
+     * @param buildNumber
+     * @param buildName
+     * @throws IOException
+     */
+    private void finalize(@NotNull File allureReportDir, int buildNumber, String buildName) throws IOException {
+        ////////////////////
+        // Update Report Name (It is the way now)
+        Path widgetsJsonPath = Paths.get(allureReportDir.getAbsolutePath()).resolve("widgets").resolve("summary.json");
+        Gson gson = new Gson();
```

Comment:
Use ObjectMapper to write object into file

Reply:
done